src/clj_ldap/client.clj

```diff
@@ -661,15 +664,16 @@ returned either before or after the modifications have taken place."
    a map that can contain:
       :pre-read     Indicates the attributes that should be read before deletion
       :proxied-auth The dn:<dn> or u:<uid> to be used as the authorization
-                    identity when processing the request."
+                    identity when processing the request.
+      :recursive    Whether or not to recursively delete the subtree"
```

Comment:
Would you mind renaming the option to :delete-subtree to convey the use of the control, as opposed to having someone think we are manually carrying out a recursive delete?

Can you tack on to the option description that the Subtree Delete Control, specified by 1.2.840.113556.1.4.805, must be supported by the server?